src/app/components/Labels/ActionsLabelToolbar.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import {
+    PrimaryButton,
+    Select,
+    Icon,
+    useModal,
+    useApiWithoutResult,
+    useEventManager,
+    useNotifications
+} from 'react-components';
+import { createLabel } from 'proton-shared/lib/api/labels';
+import { noop } from 'proton-shared/lib/helpers/function';
+
+import EditLabelModal from '../../containers/Labels/modals/Edit';
+
+function ActionsLabelToolbar({ label, onAdd, onSort, className }) {
+    const { call } = useEventManager();
+    const { createNotification } = useNotifications();
+    const { request, loading } = useApiWithoutResult(createLabel);
+    const [type, setType] = useState('');
+    const { isOpen: isOpenModal, open: openModal, close: closeModal } = useModal();
+
+    const handleClickAdd = (type) => () => {
+        setType(type);
+        openModal();
+    };
+    const handleCloseModal = closeModal;
+
+    const handleSubmitModal = async (label = {}) => {
+        const { Label } = await request(label);
+        call();
+        createNotification({
+            text: c('label/folder notification').t`${label.Name} created`
```

Comment:
Take the value coming from the API.
```suggestion
            text: c('label/folder notification').t`${Label.Name} created`
```